hosted_control_planes/hosted-control-planes-release-notes.adoc

```diff
@@ -0,0 +1,200 @@
+
+[id="hcp-4-17-taints-tolerations_{context}"]
+=== Custom taints and tolerations (Technology Preview)
+For {hcp} on {VirtProductName}, you can now apply tolerations to hosted control plane pods by using the `hcp` CLI `-tolerations` argument or by using the `hc.Spec.Tolerations` API file. This feature is available as a Technology Preview feature. For more information, see xref:../hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc#hcp-virt-taints-tolerations_hcp-distribute-workloads[Custom taints and tolerations].
+
+[id="hcp-4-17-nvidia-gpu_{context}"]
+=== Support for NVIDIA GPU devices on {VirtProductName} (Technology Preview)
+For {hcp} on {VirtProductName}, you can attach one or more NVIDIA graphics processing unit (GPU) devices to node pools. This feature is available as a Technology Preview feature. For more information, see xref:../hosted_control_planes/hcp-manage/hcp-manage-virt.adoc#hcp-virt-attach-nvidia-gpus-np-api[Attaching NVIDIA GPU devices by using the NodePool resource].
+
+[id="hcp-4-17-tenancy-aws_{context}"]
+=== Support for tenancy on {aws-short}
+When you create a hosted cluster on {aws-short}, you can indicate whether the EC2 instance should run on shared or single-tenant hardware. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-aws-deploy-hc_hcp-deploy-aws[Creating a hosted cluster on {aws-short}].
+
+[id="hcp-4-17-_{context}"]
+=== Support for {product-title} versions in hosted clusters
+You can deploy a range of supported {product-title} versions in a hosted cluster. For more information, see xref:../hosted_control_planes/hcp-updating.adoc#hcp-supported-ocp-versions_hcp-updating[Supported {product-title} versions in a hosted cluster].
+
+[id="hcp-4-17-virt-dc_{context}"]
+=== {hcp-capital} on {VirtProductName} in a disconnected environment is Generally Available
+In this release, {hcp} on {VirtProductName} in a disconnected environment is Generally Available. For more information, see xref:../hosted_control_planes/hcp-disconnected/hcp-deploy-dc-virt.adoc#hcp-deploy-dc-virt[Deploying {hcp} on {VirtProductName} in a disconnected environment].
+
+[id="hcp-4-17-multi_{context}"]
+=== {hcp-capital} for an ARM64 {product-title} cluster on {aws-short} is Generally Available
+In this release, {hcp} for an ARM64 {product-title} cluster on {aws-short} is Generally Available. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-enable-arm-amd_hcp-deploy-aws[Running hosted clusters on an ARM64 architecture].
+
+[id="hcp-4-17-aws_{context}"]
+=== {hcp-capital} for an {product-title} cluster on {aws-short} is Generally Available
+In this release, {hcp} for an {product-title} cluster on {aws-short} is Generally Available. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-enable-arm-amd_hcp-deploy-aws[Deploying hosted clusters on an {aws-short} architecture].
+
+[id="hcp-4-17-non-bm_{context}"]
```

Comment:
@xenolinux - I thought non-bare-metal was still a Technology Preview feature. If that is not the case, please let me know.

Reply:
Correct. Sorry, I got it confused it with bare metal. non-bm is still TP.